src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -123,7 +123,7 @@ public void testUserApi() throws Exception {
             new Header[0]);
```

Comment:
on line 121 comment 
Associating with  role is not allowed if roles-mapping is hidden